cmd/juju/status/status.go

```diff
@@ -401,29 +400,43 @@ func (c *statusCommand) runStatus(ctx *cmd.Context) error {
 	return nil
 }
 
-// clearScreen removes any character from the terminal
-// using ANSI scape characters.
-func clearScreen() {
-	fmt.Printf("\u001Bc")
-}
-
 func (c *statusCommand) Run(ctx *cmd.Context) error {
 	defer c.close()
 
 	if c.watch != 0 {
-		clearScreen()
-	}
+		//  Get list of all args, cut off '--watch' flag from 'juju status'
+		var jujuStatusArgsWithoutWatch []string
+		var watchValue string
+		for i := range os.Args {
+			if os.Args[i] == "--watch" {
+				watchValue = os.Args[i+1]
+				jujuStatusArgsWithoutWatch = append(os.Args[:i], os.Args[i+2:]...)
+				break
+			}
+		}
```

Comment:
this makes sense to me, but I think it would be clearer as a standalone function.
```
jujuStatusArgs := c.statusArgsWithoutWatch()
```